service/src/main/kotlin/io/provenance/api/domain/usecase/cee/execute/ExecuteContractBatch.kt

```diff
@@ -66,7 +77,9 @@ class ExecuteContractBatch(
                         log.info("Successfully processed batch $index of ${chunkedSignedResult.size}")
                     },
                     onFailure = {
-                        errors.add(it)
+                        responses.add(
+                            ContractExecutionErrorResponse(it.toPrettyJson())
```

Comment:
Same with the error class - it's provenance transaction execution error rather than p8e contract execution error